src/django/api/templates/mail/claim_facility_revocation_body.html

```diff
@@ -38,6 +38,6 @@
         <p>
```

Comment:
On line 35 we include the info@ email in the body text. I would suggest changing it to data@ to harmonize with the from address.

Reply:
https://github.com/open-apparel-registry/open-apparel-registry/pull/2373/commits/42105b30aedb53f350ed5446a027982f645fd1f2